bin/make_db

```diff
@@ -1,5 +1,35 @@
-"""Initialize the dev environment's DB."""
 #!/usr/bin/env python3
-import sys
+"""Initialize the dev environment's DB."""
+import alembic.command
+import alembic.config
 from pyramid.paster import bootstrap
-bootstrap("conf/development.ini")
+from sqlalchemy import text
+from sqlalchemy.exc import ProgrammingError
+
+from via.db import Base, create_engine
+
+
+def is_stamped(engine) -> bool:
+    """Return True if the DB is stamped with an Alembic revision ID."""
+    with engine.connect() as connection:
+        try:
+            if connection.execute(text("select * from alembic_version")).first():
+                return True
+        except ProgrammingError:
+            pass
+
+    return False
+
+
+def main():
+    with bootstrap("conf/development.ini") as env:
+        settings = env["registry"].settings
+        engine = create_engine(settings)
+
+        if not is_stamped(engine):
+            Base.metadata.create_all(engine)
+            alembic.command.stamp(alembic.config.Config("conf/alembic.ini"), "head")
```

Comment:
As I mentioned on the other PR I reckon either here or in the Makefile
we should run the migrations as part of the `make db` process 

https://github.com/hypothesis/lms/blob/main/Makefile#L18

I reckon with that we'll get a "it just works" experience for dev that are not making changes here often and just want to have the service running.

Reply:
Via [does run the migrations in `Makefile`](https://github.com/hypothesis/via/blob/c97bceaf640c91493f394ab0340e6c6c9cc68f32/Makefile#L18). I think `make db` has always done this for all our apps?

It may be a bit confusing that there's a `bin/make_db` script, but calling that script is only one of two commands that `make db` runs. May want to move the `alembic uprade` command into the `make_db` script somehow